setup.py

```diff
@@ -23,10 +23,10 @@
     install_requires=[
         "grpcio==1.37.1",
         "grpcio-tools==1.37.1",
-        "ujson>=2.0.0,<=5.1.0",
+        "ujson>=1.3.5,<=5.1.0",
```

Comment:
I'm afraid ujson doesn't have version 1.3.5, please change back to 2.0.0

Here's a ref https://pypi.org/project/ujson/#history